AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -348,9 +348,9 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         }
     }
 
-    private func notifyStartSucceeded() {
+    private func notifyStartSucceeded(newAudioStatus: MeetingSessionStatusCode) {
         meetingStatsCollector.updateMeetingStartTimeMs()
-        eventAnalyticsController.publishEvent(name: .meetingStartSucceeded)
+        eventAnalyticsController.publishEvent(name: .meetingStartSucceeded, attributes: [EventAttributeName.meetingStatus: newAudioStatus])
```

Comment:
Won't the status be always `OK` for success?

Reply:
Yes, but it makes sense to provide it. `meetingStatus` is independent of the operation so it makes searching ingestion logs more intuitive.